src/reducers/profile-view.js

```diff
@@ -110,15 +116,18 @@ function symbolicationStatus(
   }
 }
 
-function viewOptionsPerThread(state: ThreadViewOptions[] = [], action: Action) {
+function viewOptionsPerThread(
+  state: ThreadViewOptions[] = EMPTY_THREAD_VIEW_OPTIONS,
+  action: Action
+) {
   switch (action.type) {
     case 'RECEIVE_PROFILE_FROM_ADDON':
     case 'RECEIVE_PROFILE_FROM_STORE':
     case 'RECEIVE_PROFILE_FROM_URL':
     case 'RECEIVE_PROFILE_FROM_FILE':
       return action.profile.threads.map(() => ({
-        selectedCallNodePath: [],
-        expandedCallNodePaths: [],
+        selectedCallNodePath: EMPTY_CALL_NODE_PATH,
+        expandedCallNodePaths: EMPTY_CALL_NODE_PATH_LIST,
```

Comment:
After closer exanimation, I think these ones are useful because they're used in accessors. 👍